.github/workflows/daily-builds.yml

```diff
@@ -0,0 +1,12 @@
+name: Daily builds
+
+on:
+  workflow_dispatch:
+
+jobs:
+  checkbox-core-snap-daily:
+    uses: ./.github/workflows/checkbox-core-snap-daily-builds.yml
+  checkbox-snap-daily:
+    uses: ./.github/workflows/checkbox-snap-daily-builds.yml
+  checkbox-deb-daily:
+    uses: ./.github/workflows/deb-daily-builds.yml
```

Comment:
Remove this please, accidentally committed?

Reply:
Oops, not sure how that made it there. Fixed.